open_feature/provider/no_op_provider.py

```diff
@@ -1,8 +1,8 @@
 import typing
 
 from open_feature.evaluation_context.evaluation_context import EvaluationContext
-from open_feature.flag_evaluation.flag_evaluation_details import FlagEvaluationDetails
 from open_feature.flag_evaluation.reason import Reason
+from open_feature.flag_evaluation.resolution_details import FlagResolutionDetails
```

Comment:
to armonize the import order, we could introduce [black](https://pypi.org/project/black/)

Reply:
I ran the file through black and nothing changed 🤷🏼

Reply:
Sorry, I didn't explain properly. What I was suggesting is to have a check that is run on a PR level to make sure the code is formatted properly. Something like the following: https://github.com/open-telemetry/build-tools/blob/main/.github/workflows/semconvgen.yml#L28

Reply:
Ah yes that makes sense, but probably in a separate issue :)